lib/glueby/contract/token_tx_builder.rb

```diff
@@ -0,0 +1,201 @@
+# frozen_string_literal: true
+
+module Glueby
+  module Contract
+    module TokenTxBuilder
+      def receive_address(wallet:)
+        wallet.receive_address
+      end
+
+      # Create new public key, and new transaction that sends TPC to it
+      def create_funding_tx(wallet:, amount:, script: nil, fee_provider: FixedFeeProvider.new)
+        tx = Tapyrus::Tx.new
+        fee = fee_provider.fee(tx)
```

Comment:
The fee calculation depends on whole tx size, so fee should be calculate after construct the transaction.

Reply:
12473cc58b11cef298de6887112b32aaa8adb790

Add method to create dummy tx and use it for calculating the fee.